src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php

```diff
@@ -93,7 +93,9 @@ public function configure(Event $event = null)
         }
         if (!$this->exceptionHandler) {
             if ($event instanceof KernelEvent) {
-                $this->exceptionHandler = array($event->getKernel(), 'terminateWithException');
+                if (is_callable(array($event->getKernel(), 'terminateWithException'))) {
```

Comment:
yes, but I'd say we don't care, we already use method_exists in the code base and that doesn't cause any issue I think